awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -62,6 +61,38 @@ function JobTemplateForm({
     Boolean(template?.host_config_key)
   );
 
+  const formikBag = useFormikContext();
+  const formikValues = formikBag.values;
```

Comment:
I'd get the value off the first or second (meta) items of the array returned from the useField call